arm-datalake-store/account/2016-11-01/swagger/account.json

```diff
@@ -0,0 +1,1311 @@
+    },
+    "DataLakeStoreAccountListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/DataLakeStoreAccount"
+          },
+          "description": "the results of the list operation"
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the link (url) to the next page of results."
+        },
+        "count": {
+          "type": "integer",
+          "readOnly": true,
+          "format": "int64",
+          "description": "the total count of results that are available, but might not be returned in the current page."
+        }
+      },
+      "description": "Data Lake Store account list information response."
+    },
+    "ErrorDetails": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the HTTP status code or error code associated with this error"
+        },
+        "message": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the error message localized based on Accept-Language"
+        },
+        "target": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the target of the particular error (for example, the name of the property in error)."
+        }
+      },
+      "description": "Data Lake Store error details information"
+    },
+    "InnerError": {
+      "properties": {
+        "trace": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the stack trace for the error"
+        },
+        "context": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the context for the error message"
+        }
+      },
+      "description": "Data Lake Store inner error information"
+    },
+    "Error": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the HTTP status code or error code associated with this error"
+        },
+        "message": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the error message to display."
+        },
+        "target": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the target of the error."
+        },
+        "details": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ErrorDetails"
+          },
+          "description": "the list of error details"
+        },
+        "innerError": {
+          "$ref": "#/definitions/InnerError",
+          "readOnly": true,
+          "description": "the inner exceptions or errors, if any"
+        }
+      },
+      "description": "Data Lake Store error information"
+    },
+    "AzureAsyncOperationResult": {
```

Comment:
No, I think it was a hold-over from hyak. It can be removed since there is a new way to handle LROs